### ISSUE TITLE
reduce sidebar when not hovered

### ISSUE THREAD

**Issue by rinaldi6109** [eoger/tabcenter-redux#395]
Is it possible to shrink or hide the sidebar when not hovered by the mouse and enlarge it only on mouse hovering?

**Comment by Keith94** [comment]
Yes, but only with hacks. See https://github.com/eoger/tabcenter-redux/wiki/Custom-CSS-Tweaks#floating-auto-shrinking-sidebar-103
